packages/stargate/src/signingstargateclient.ts

```diff
@@ -87,6 +89,11 @@ export const defaultRegistryTypes: ReadonlyArray<[string, GeneratedType]> = [
   ["/cosmos.staking.v1beta1.MsgDelegate", MsgDelegate],
   ["/cosmos.staking.v1beta1.MsgEditValidator", MsgEditValidator],
   ["/cosmos.staking.v1beta1.MsgUndelegate", MsgUndelegate],
+  ["/cosmos.authz.v1beta1.MsgGrant", MsgGrant],
+  ["/cosmos.authz.v1beta1.MsgExec", MsgExec],
+  ["/cosmos.authz.v1beta1.MsgRevoke", MsgRevoke],
+  ["/cosmos.feegrant.v1beta1.MsgGrantAllowance", MsgGrantAllowance],
+  ["/cosmos.feegrant.v1beta1.MsgRevokeAllowance", MsgRevokeAllowance],
```

Comment:
Could you order them alphabetically in the list (at least the new entrief if the rest is not sortted)?